### DISCUSSION TITLE
Changing EEP Profile A5-07-01 for Eltako FTTB/Piontek Tracker

### DISCUSSION THREAD

**Discussion by memooox3** [mak-gitdev/HA_enoceanmqtt#82] [Request]
Hi,

I have two Eltako FTTB presence Tracker for Tracking the presence of my wife and me. (Used as keychain)
This device Push every Minute an presence Telegramm, but no Telegramm when the device is outside the enocean range(=Not at Home). 
So the pir -entity will Always stay as presence.
Is it possible to automatically set the entity after a specific time to unavailable?

![Screenshot_2023-08-28-11-08-44-881_com android chrome](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/40836316/6823cb61-ad0a-4493-adbd-e16ad18e7310)

**Comment by mak-gitdev** [comment]
Hi @memooox3,

Not sure I completely understood your request but HA_enoceanmqtt is only intended to exhibit the exact status of an EnOcean device.
Hence, I cannot add a custom behavior to device entities.

But you can easily achieve what you want to do at HA level by using a [template sensor](https://www.home-assistant.io/integrations/template/) on top of your e2m_xxx entities.

**EDIT:** You could also use a [custom mapping file](https://github.com/mak-gitdev/HA_enoceanmqtt/wiki/Examples#using-custom-mappingyaml-andor-eepxml-files) so that you can define your entities as you want but this is more cumbersome.

**Reply by memooox3** [reply]
Hi,
I'm new to HA, so I hadn't heard about the templates or their possibilities.

With an sensor template I have a working solution now, thank you for your help.
